tests/test_data/artists/gosha.json

```diff
@@ -3,7 +3,8 @@
 	"socials": {
 		"instagram": "https://gosha_inst.com",
 		"youtube": "https://gosha_youtube.com",
-		"spotify": "https://gosha_spotify.com"
+		"spotify": "https://gosha_spotify.com",
+		"wiki": "https://en.wikipedia.org/wiki/Gosha_woman"
```

Comment:
That is very insulting and inappropriate, I like it

Reply:
that's the real wikipedia page though